autopts/wid/bap.py

```diff
@@ -305,26 +316,27 @@ def hdl_wid_114(params: WIDParams):
 
     presentation_delay = 40000
     subgroups = 1
-    broadcast_id = btp.bap_broadcast_source_setup(
-        streams_per_subgroup, subgroups, coding_format, vid, cid,
-        codec_ltvs_bytes, *qos_config, presentation_delay)
 
-    stack = get_stack()
-    stack.bap.broadcast_id = broadcast_id
+    broadcast_id = btp.bap_broadcast_source_setup(streams_per_subgroup, subgroups,
+                                                  coding_format, vid, cid, codec_ltvs_bytes,
+                                                  *qos_config, presentation_delay,
+                                                  broadcast_id)
 
     btp.bap_broadcast_adv_start(broadcast_id)
 
     btp.bap_broadcast_source_start(broadcast_id)
 
     data = bytearray([j for j in range(0, 41)])
 
-    for i in range(1, 100):
+    for i in range(1, 10):
```

Comment:
This change won't affect Zephyr, so suggest to revert

Reply:
Done, only changed it temporary for quicker runs while testing, but forgot to change it back